PENDING.md

```diff
@@ -46,9 +46,14 @@ FEATURES
   * [\#2730](https://github.com/cosmos/cosmos-sdk/issues/2730) Add tx search pagination parameter
   * [\#3027](https://github.com/cosmos/cosmos-sdk/issues/3027) Implement
   `query gov proposer [proposal-id]` to query for a proposal's proposer.
+  * [\#3198](https://github.com/cosmos/cosmos-sdk/issues/3198) New `keys add --multisig` flag to store multisig keys locally.
```

Comment:
Generally when I have multi-item log entries like this for a single PR/issue, I just list the PR/issue once and then use bullet points for each entry.